src/client.ts

```diff
@@ -903,6 +910,19 @@ export class ConnectionImpl {
 
         case chars.$Z: {
           this._parseSyncMessage();
+
+          if (
+            !(this.sock instanceof tls.TLSSocket) &&
+            // @ts-ignore
+            typeof Deno === "undefined" &&
+            this.protocolVersion >= [0, 11]
```

Comment:
This is, unfortunately, an incorrect way -- JS would cast the arrays to strings and compare strings. Most of the times that will work until it doesn't one day. Instead, do what you did before -- have two functions to do the version comparisons, but on two-element tuples, not objects.

Reply:
changed